kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala

```diff
@@ -249,7 +249,8 @@ class ServiceControlCliArguments(args: Seq[String], env: Map[String, String] = s
     }
 
     try {
-      cliArgs = cliArgs.copy(host = InetAddress.getByName(cliArgs.host).getCanonicalHostName)
+      InetAddress.getByName(cliArgs.host)
```

Comment:
Because `kyuubi.frontend.connection.url.use.hostname` is configured to determine whether the node registered to zookeeper is hostname or ip, would it be better to use the original value passed in with `--host` here?

Reply:
> Because `kyuubi.frontend.connection.url.use.hostname` is configured to determine whether the node registered to zookeeper is hostname or ip, would it be better to use the original value passed in with `--host` here?

When we configure `kyuubi.frontend.thrift.binary.bind.host`, the `kyuubi.frontend.connection.url.use.hostname` config does not seem to take effect, we use the original host argument will be more flexible.

Reply:
The server is configured with `kyuubi.frontend.thrift.binary.bind.host`, so `kyuubi.frontend.connection.url.use.hostname` does not take effect.
https://github.com/apache/incubator-kyuubi/blob/deb0e6203b5b56a1c61162f793d3601be0395118/kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala#L109-L119


Because there may be the ip used by the server and the hostname used by the engine, it is sometimes embarrassing to convert `--host`.